snippets/get_transaction.py

```diff
@@ -0,0 +1,31 @@
+"""A snippet that walks us through getting a transaction."""
```

Comment:
```suggestion
"""How to see a validated transaction"""
```
(Or something else more active + about the lesson rather than the implementation)